features/emscripten/build.cmd

```diff
@@ -0,0 +1,5 @@
+@echo off
```

Comment:
The build script needs to be `build.bat`

Reply:
renamed